app/views/posts/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Posts#index</h1>
```

Comment:
- [ ] Good job creating an index and show views for posts, but you need to do the same for users' index and show as well 
![image](https://user-images.githubusercontent.com/62705947/229642277-2e77db16-fbdf-48ea-b90a-7fd22b7e5163.png)